src/cluster/placement/algo/sharded_test.go

```diff
@@ -1306,6 +1306,42 @@ func TestBalanceShardsForShardedWhenImbalanced(t *testing.T) {
 	assert.Equal(t, expectedInstances, balancedPlacement.Instances())
 }
 
+func TestInitialPlacementIsBalanced(t *testing.T) {
```

Comment:
Do you think it would be useful to apply property based testing in this case? https://pkg.go.dev/testing/quick
To be more confident we generate well balanced shards.

Reply:
As a matter of fact we are using `github.com/leanovate/gopter`.